client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -629,6 +664,72 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
         setOrderBy(orderByValue);
     }
 
+    const getInvestigationsByGroupId = async (groupId: string) => {
+        const investigationsByGroupIdLogger = logger.setup({
+            workflow: 'get investigations by group id',
+            user: user.id,
+            investigation: epidemiologyNumber
+        });
+
+        investigationsByGroupIdLogger.info('send get investigations by group id request', Severity.LOW);
+        setIsLoading(true)
+        !allGroupedInvestigations.get(groupId) && 
+        await axios.get('/groupedInvestigations/investigations/' + groupId).then((result) => {
```

Comment:
I don't think you need to name your route investigations
keep it empty and just pass groupId